test/powershell/engine/Basic/DefaultConfirmImpact.Tests.ps1

```diff
@@ -0,0 +1,256 @@
+Describe 'Default Cmdlet ConfirmImpact Ratings' -Tags 'CI' {
+    BeforeAll {
+        $moduleList = @(
+            "Microsoft.PowerShell.Utility"
+            "Microsoft.PowerShell.Management"
+            "Microsoft.PowerShell.Security"
+            "Microsoft.PowerShell.Host"
+            "Microsoft.PowerShell.Diagnostics"
+            "PSWorkflow"
+            "Microsoft.WSMan.Management"
+            "Microsoft.PowerShell.Core"
+        )
+        Import-Module -Name $moduleList -ErrorAction SilentlyContinue
```

Comment:
Actually, you shouldn't have to explicitly have this list of modules and import them as auto-module loading will take care of that for you

Reply:
It wasn't. 🤷‍♂️ 

Not sure why, though.

Reply:
This did not work in DefaultsCommands.Tests.ps1 too.